### ISSUE TITLE
No member completion list shown on constructor parameters

### ISSUE THREAD

**Issue by cpyrgas** [X-Sharp/XSharpPublic#652]
```
CLASS Foo
CONSTRUCTOR(n AS INT, c AS STRING)
n: // no member completion
c: // no member completion
END class

```

**Comment by RobertvanderHulst** [comment]
Do you want me to fix this before releasing 2.8a ?

**Comment by cpyrgas** [comment]
No, no, I just logged it for a next version. But I see you have already fixed it now :)
